setup.cfg

```diff
@@ -1,60 +1,41 @@
 [metadata]
-name = packitos
-url = https://github.com/packit-service/packit
-description = A set of tools to integrate upstream open source projects into Fedora operating system.
+name = packit-service
+url = https://github.com/packit-service/packit-service
+description = A service to integrate upstream open source projects into Fedora operating system.
 long_description = file: README.md
 long_description_content_type = text/markdown
 author = Red Hat
 author_email = user-cont-team@redhat.com
 license = MIT
 license_file = LICENSE
 classifiers =
-    Development Status :: 4 - Beta
-    Environment :: Console
     Intended Audience :: Developers
     License :: OSI Approved :: MIT License
     Operating System :: POSIX :: Linux
     Programming Language :: Python
-    Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Topic :: Software Development
-    Topic :: Utilities
 keywords =
     git
     packaging
     fedora
     rpm
     dist-git
+    packit
```

Comment:
So this always installs released `packit` from PyPI.
What if we want to continuously build & deploy packit from master to our staging instance?

Since this is a more broader topic, I've filled #3 to discuss it there.